packages/magento-cart-email/EmailForm/EmailForm.tsx

```diff
@@ -69,6 +69,7 @@ const EmailFormBase = React.memo<EmailFormProps>((props) => {
           })}
           InputProps={{
             autoComplete: 'email',
+            autoFocus: true,
```

Comment:
Do we always want to autoFocus this? Maybe better to pass this by a prop, seems a configuration specific to the place where the component is used.